src/main/java/teammates/common/datatransfer/FeedbackNumericalScaleQuestionDetails.java

```diff
@@ -467,12 +470,12 @@ public String getQuestionResultStatisticsCsv(
         df.setMaximumFractionDigits(5);
         df.setRoundingMode(RoundingMode.DOWN);
   
-        String csvHeader = "";
-        csvHeader += "Team, Recipient, Average, Minimum, Maximum";
-        csvHeader += showAvgExcludingSelf ? ", Average excluding self response" : "";
-        csvHeader += Const.EOL;
+        StringBuilder csvHeader = new StringBuilder(100);
+        csvHeader.append("Team, Recipient, Average, Minimum, Maximum")
+                 .append(showAvgExcludingSelf ? ", Average excluding self response" : "")
+                 .append(Const.EOL);
```

Comment:
I think it's safe for `csvHeader` to be assigned the old fashioned way, `String csvHeader = ... + ... + ...`?